parser/tests/Create.test

```diff
@@ -139,6 +139,74 @@ CREATE TABLE test (attr INT) CHECK(attr>0)
                              ^
 ==
 
+CREATE TABLE test (attr INT) PARTITION BY HASH(attr) PARTITIONS 4
+--
+CreateTableStatement[relation_name=test]
++-attribute_list=
+| +-AttributeDefinition[name=attr,type=Int]
++-partition_clause=
+  +-PartitionClause[partition_type=0]
+    +-Number of Partitions=NumericLiteral[numeric_string=4,float_like=false]
+    +-attribute_name_list=
+      +-String[value=attr]
+==
+
```

Comment:
This PR adds support only for hash-based partition scheme and hence I didn't add the RANGE partition test case here. That will be part of a separate PR. Although the lexer support has been added for the RANGE keyword, parser support hasn't been added yet.

Also, can you please contact the PR owner before cancelling any builds in the future? Thanks!